tests/test_connectivity.py

```diff
@@ -0,0 +1,24 @@
+"""
+Unit tests for basic connectivity in CashCtrlLedger.
+These tests check that essential data is present and not empty.
+"""
+
+import pytest  # Using pytest framework for testing
+from cashctrl_ledger import CashCtrlLedger
+
+@pytest.fixture
```

Comment:
Is there any advantage of using a picture in this simple case?

Calling `CashCtrlLedger()` directly instead of the fixture would IMHO improve code readability.